### DISCUSSION TITLE
Notifications failing at `notify()`

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#9582] [General - Components / Directives / etc] [answered]
I am trying to fire a simple [notification ](https://next.quasar.dev/quasar-plugins/notify) in Quasar 2:

```
setup() {
  const $q = useQuasar()
   $q.notify('hello')
}
```

This fails with:

    Uncaught TypeError: $q.notify is not a function

This is a ~~UMD~~ `vue-cli` (EDIT: this is where the confusion came from)  application that works fine without these two lines - I do not really know where to go from there as the docs say that there is nothing special to configure before using it.

Interestingly, my IDE is suggesting me `notify()` when typing `$q.` so  at least at that level is it well recognized.

**Comment by metalsadman** [comment, marked as answer]
@wsw70 https://next.quasar.dev/quasar-plugins/notify#installation

**Reply by wsw70** [reply]
Yes, I know, and it says

```
/*
 * No installation step is necessary.
 * It gets installed by default.
 */
```

**Reply by metalsadman** [reply]
If you are using UMD, then you don't need to use `useQuasar` check https://next.quasar.dev/start/umd#quasar-global-object.

ie. `Quasar.Notify.create('Hi and welcome!')`

**Reply by wsw70** [reply]
Thanks for the information. I got quite confused with the variants and thought I was using UMD when it was actually vue-cli....

I added the configuration for vue-cli and now it is fine, thanks!
I also updated the question with that info.
